cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,26 @@
+/// <reference types="cypress" />
+
+const { generateUser } = require('../support/generate');
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
+      
+    cy.visit('https://react-redux.realworld.io/#/register');
+
+    cy.get(':nth-child(1) > .form-control')
+      .type(user.username);
+    
+    cy.get(':nth-child(2) > .form-control')
+      .type(user.email);
+
+      cy.get(':nth-child(3) > .form-control')
```

Comment:
try to find better selectors

Reply:
do you mean writing this with placeholders would be better?